--- install_xite	Thu Jan 23 19:27:37 2003
+++ install_xite_Debian	Tue Jan 28 13:54:46 2003
@@ -969,7 +969,7 @@
 else
 	# Don't rebuild documentation (man pages and hypertext)
 	if test "$XITE_HOSTTYPE" != "NT4"; then
-		makexite_command="$makexite -v -c -a $XITE_HOSTTYPE -i -l $dso -e $local_opt -xite $XITE_HOME"
+		makexite_command="$makexite -v -c -d -i -l $dso -e $local_opt -xite $XITE_HOME"
 	else
 		makexite_command="$makexite -v -c -a $XITE_HOSTTYPE -xite $XITE_HOME"
 	fi
