--- etc/.install/makeMakefile.orig	2005-11-29 12:12:58.000000000 +0100
+++ etc/.install/makeMakefile	2006-02-20 11:21:37.621673296 +0100
@@ -567,15 +567,15 @@
 DIRENT          = 
 
 CFLAGS_ALWAYS   = -D$(XITE_HOSTTYPE) $(DEBUG) -Dinstring=__instring \
                   $(REGEX) $(DIRENT) -m32
 
-OPTIM           = -O
+OPTIM           = -O3
 OPTIM_GNU       = -O3
 CDEBUG          = -g
 CC_DSO_FLAGS    = -fPIC
-CARCH_OPT       = 
+CARCH_OPT       = -march=pentium4 -mfpmath=sse -msse2
 
 EOF_Linux
 
 elif test "$XITE_HOSTTYPE" = "Debian"
 then
